python/vineyard/data/utils.py

```diff
@@ -41,7 +41,7 @@ def normalize_dtype(dtype, dtype_meta=None):
     if dtype in ['u64', 'uint64', 'uint64_t']:
         return np.dtype('uint64')
     if dtype in ['float', 'float32']:
-        return np.dtype('float')
+        return np.dtype('float32')
```

Comment:
I think this should be if dtype == 'float': return np.dtype('float')  if dtype == 'float32': return np.dtype('float32') ?

Reply:
Will Change the code accordingly